### PR TITLE
feat:OS CLI output part III - count issues +test

### PR DESCRIPTION
**Pull request by florindumitrascu**
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

#### What does this PR do?
Improve snyk-open-source CLI output as follows:
- Provide counts where possible (e.g. Upgrade package@2.3.20 to package@2.5.26 to fix 6 issues)

#### Where should the reviewer start?
src/lib/formatters/remediation-based-format-issues.ts

#### How should this be manually tested?
Run 'snyk test' command on a local project and compare output formatting with the attached screenshot

#### What are the relevant tickets?
[TUN-269](https://snyksec.atlassian.net/browse/TUN-269)

#### Screenshots
Screenshot 2022-07-27 at 16 02 15

Screenshot 2022-07-27 at 16 03 06

### CODE REVIEW

File: src/lib/formatters/remediation-based-format-issues.ts [hunk 1]
```diff
@@ -161,7 +161,9 @@ function constructLicenseText(
     return [];
   }
 
-  const licenseTextArray = [chalk.bold.green('\nLicense issues:')];
+  // const licenseTextArray = [chalk.bold.green('\nLicense issues:')];
```
Comment (Avishagp) [inline src/lib/formatters/remediation-based-format-issues.ts:164]: Should this be deleted?

Comment (Avishagp) [inline src/lib/formatters/remediation-based-format-issues.ts:164]: Should this line be deleted?

Comment (florindumitrascu) [reply src/lib/formatters/remediation-based-format-issues.ts]: Sure, my bad. Fixed now.

Comment (florindumitrascu) [reply src/lib/formatters/remediation-based-format-issues.ts]: Fixed now

**Review by Avishagp — APPROVED** [review]

**Pull request closed without merging**